Frameworks/UIKit/UIView.mm

```diff
@@ -198,24 +199,68 @@ - (bool)_processGesturesForTouch:(UITouch*)touch event:(UIEvent*)event touchEven
         }
     }
 
-    // gesture priority list
-    const static id s_gesturesPriority[] = {[UIPinchGestureRecognizer class],
-                                            [UISwipeGestureRecognizer class],
-                                            [UIPanGestureRecognizer class],
-                                            [UILongPressGestureRecognizer class],
-                                            [UITapGestureRecognizer class] };
```

Comment:
You're not wrong re: the order of things (even though the iOS docs say the order in which they get serviced is undefined behaviour, it does appear to be deterministic). We don't have a test suite, since our UI automation story hasn't been fleshed out yet, so everything is manual testing for the time being. 

After a little bit of messing around with iOS, it looks like returning NO from `canPreventGestureRecognizer` and `canBePreventedByGestureRecognizer` preempts the calls to the delegate.. that is to say, you should be nesting your calls to the delegate inside a check to see if the gesture can be prevented by other gestures. 

I bring this up specifically because I'd like an avenue to keep the old behaviour while this work gets finished up, since as of this change only one gesture will be fired in most cases (where previously we'd fire all gestures). To enable that, I'd like you to nest that logic, and add a default implementation to `UIGestureRecognizer.mm` to return `NO` for `canPreventGestureRecognizer:` and `canBePreventedByGestureRecognizer:`, so all gestures fire simultaneously (unless of course someone were to subclass or add exclusion logic in their gesture handler selectors).

Thanks! Let me know if I need to clarify anything.